pkg/gds/config/config.go

```diff
@@ -38,13 +38,13 @@ type GDSConfig struct {
 }
 
 type AdminConfig struct {
-	Enabled           bool     `split_words:"true" default:"true"`
-	BindAddr          string   `split_words:"true" default:":4434"`
-	Mode              string   `split_words:"true" default:"release"`
-	Audience          string   `split_words:"true"`
-	AuthorizedDomains []string `split_words:"true"`
-	AllowOrigins      []string `split_words:"true" default:"http://localhost,http://localhost:3000,http://localhost:3001"`
-	CookieDomain      string   `split_words:"true"`
+	Enabled      bool     `split_words:"true" default:"true"`
+	BindAddr     string   `split_words:"true" default:":4434"`
+	Mode         string   `split_words:"true" default:"release"`
+	AllowOrigins []string `split_words:"true" default:"http://localhost,http://localhost:3000,http://localhost:3001"`
+	CookieDomain string   `split_words:"true"`
+	Audience     string   `split_words:"true"`
```

Comment:
This `Audience` configuration refers to the token audience claim verification not the Google credential audience, which has been moved to the OauthConfig.